### PR TITLE
Add an overview of the API endpoints in the docs

### PR DESCRIPTION
**Pull request by KevinEtienne**

### CODE REVIEW

**Comment by KevinEtienne** [comment]
@incuna/backend merge?

**Comment by adam-incuna** [comment]
Unless the list is wrong or incomplete (I don't know DUM well enough to confirm), this looks good :P

**Comment by meshy** [comment]
Might be worth explaining the modular nature of the `urls` module that provides these.

**Comment by adam-incuna** [comment]
Travis hath exploded weirdly.

screenshot from 2015-03-23 09 46 50

File: docs/index.md [hunk 1]
```diff
@@ -7,3 +7,50 @@ Documentation generated with [mkdocs.org](http://mkdocs.org).
...
+
+## API endpoints
+
+Including `user_management.api.urls` will give the following API endpoints:
+ - `auth`
+ - `password_reset`
+ - `profile`
+ - `register`
+
+If you need more control, urls are splitted across several files and can be included
```
Comment (meshy) [inline docs/index.md]: **split** ~~ted~~

**Comment by meshy** [comment]
can the urls link to the views in the views.md file?

**Comment by KevinEtienne** [comment]
@meshy [views.md](https://github.com/incuna/django-user-management/blob/ed8e647586c5a9ca48e9223aa19ab99d545c3167/docs/views.md) is not detailed well enough to link to it.

Ideally `views.md` should contain a paragraph introducing what they does in it.

**Comment by adam-incuna** [comment]
This looks good to me, although see my earlier comment about not knowing DUM well enough to say for sure :)

**Pull request merged**